src/nvlink/worker_32bit.py

```
@@ -0,0 +1,381 @@
+            self.nvlink = win32com.client.Dispatch("NVDTLabLib.NVLink")
+            self.nvlink.ParentHWnd = 0
+
+            ret = self.nvlink.NVInit("UNKNOWN")
+
+            if ret == 0:
+                self.initialized = True
+                return {"success": True, "return_code": ret}
+            else:
+                return {"success": False, "return_code": ret, "error": f"NVInit failed with code {ret}"}
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)